src/App.tsx

```diff
@@ -1,26 +1,270 @@
+        {todos.length > 0 && (
+          <footer className="todoapp__footer" data-cy="Footer">
+            <span className="todo-count" data-cy="TodosCounter">
+              {`${todos.filter(t => !t.completed).length} items left`}
+            </span>
+
+            {/* Active link should have the 'selected' class */}
+            <nav className="filter" data-cy="Filter">
+              <a
+                href="#/"
+                className={classNames('filter__link', {
+                  selected: sortTodos === 'All',
+                })}
+                data-cy="FilterLinkAll"
+                onClick={() => setSortTodos('All')}
+              >
+                All
+              </a>
+
+              <a
+                href="#/active"
+                className={classNames('filter__link', {
+                  selected: sortTodos === 'Active',
+                })}
+                data-cy="FilterLinkActive"
+                onClick={() => setSortTodos('Active')}
+              >
+                Active
+              </a>
+
+              <a
+                href="#/completed"
+                className={classNames('filter__link', {
+                  selected: sortTodos === 'Completed',
+                })}
+                data-cy="FilterLinkCompleted"
+                onClick={() => setSortTodos('Completed')}
+              >
+                Completed
+              </a>
+            </nav>
+
+            {/* this button should be disabled if there are no completed todos */}
+            <button
+              type="button"
+              className="todoapp__clear-completed"
+              data-cy="ClearCompletedButton"
+              disabled={!todos.some(t => t.completed)}
+              onClick={deleteAllCompleted}
+            >
+              Clear completed
+            </button>
+          </footer>
+        )}
+      </div>
+      {/* DON'T use conditional rendering to hide the notification */}
+      {/* Add the 'hidden' class to hide the message smoothly */}
+      <div
+        data-cy="ErrorNotification"
+        className={classNames(
+          'notification is-danger is-light has-text-weight-normal',
+          { hidden: error === null },
+        )}
+      >
+        <button data-cy="HideErrorButton" type="button" className="delete" />
+        {/* show only one message at a time */}
+        {error === 'LOAD' && (
+          <>
+            Unable to load todos
+            <br />
+          </>
+        )}
+        {error === 'TITLE' && (
+          <>
+            Title should not be empty
+            <br />
+          </>
+        )}
+        {error === 'ADD' && (
+          <>
+            Unable to add a todo
+            <br />
+          </>
+        )}
+        {error === 'DELETE' && (
+          <>
+            Unable to delete a todo
+            <br />
+          </>
+        )}
+        {error === 'UPDATE' && (
+          <>
+            Unable to update a todo
+            <br />
+          </>
```

Comment:
you shoud use state value here
```suggestion
        {errorMessage}
```